source/_integrations/rainbird.markdown

```diff
@@ -17,8 +16,10 @@ This `rainbird` integration allows interacting with [LNK WiFi](https://www.rainb
 
 There is currently support for the following device types within Home Assistant:
 
+- [Configuration](#configuration)
 - [Sensor](#sensor)
 - [Switch](#switch)
+- [Services](#services)
```

Comment:
Configuration and services are not devices type. Please remove this unrelated change.

Reply:
sorry about that, think my markdown linter snuck this in on me without me realizing it. I'll remove it.